broker-daemon/broker-rpc/admin-service/register.js

```diff
@@ -15,15 +9,24 @@ const REGISTER_URL = 'https://sparkswap.com/register/'
  * @param {Object} responses
  * @param {Function} responses.RegisterResponse - constructor for RegisterResponse messages
  * @returns {RegisterResponse}
+ * @throws {Error} Unable to find registration url
  */
-async function register ({ relayer, logger }, { RegisterResponse }) {
+async function register ({ relayer, logger, network }, { RegisterResponse }) {
   const publicKey = relayer.identity.pubKeyBase64
 
   // Currently we don't do anything with this entityId but we will need it in the future
   const { entityId } = await relayer.adminService.register({ publicKey })
 
   logger.info('Successfully registered Broker with relayer', { entityId })
-  const url = `${REGISTER_URL}${entityId}`
+
+  const registerUrl = registerUrls[network]
+
+  if (!registerUrl) {
+    throw new Error('Unable to find registration url, please check your broker configuration')
```

Comment:
`Could not find registration url for network ${network}, please check broker configuration`